src/main/java/sg/ncl/testbed_interface/SignUpMergedForm.java

```diff
@@ -1,5 +1,8 @@
 package sg.ncl.testbed_interface;
 
+import org.hibernate.validator.constraints.NotEmpty;
+import org.hibernate.validator.constraints.Range;
+
 import javax.validation.constraints.AssertTrue;
 import javax.validation.constraints.NotNull;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.validation.constraints.NotNull'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)